interfaces/Portalicious/src/app/pages/project-registration-debit-cards/project-registration-debit-cards.page.html

```diff
@@ -42,7 +42,7 @@
       <app-data-list
         [data]="walletWithCurrentCardListData()"
         data-testid="wallet-current-card-list"
-        i18n-
+        i18n
```

Comment:
```suggestion
```

Reply:
Haha, lol. 🤯 That was _totally_ a change to see if your where paying attention... 😇